src/main/java/io/openliberty/tools/intellij/liberty/lsp/LibertyConfigLanguageServer.java

```diff
@@ -33,20 +32,10 @@ public class LibertyConfigLanguageServer extends ProcessStreamConnectionProvider
     private static final Logger LOGGER = LoggerFactory.getLogger(LibertyConfigLanguageServer.class);
 
     public LibertyConfigLanguageServer() {
+        String javaHome = System.getProperty("java.home");
+        checkJavaHome(javaHome, Constants.LIBERTY_CONFIG_SERVER);
```

Comment:
The spirit of Trevor's comment is that moving the checks on Java home into super constructor changed the behaviour of this method, causing execution to continue if there's a problem with the JDK installation. checkJavaHome() should probably return a boolean which is checked in this method so that it can return as before.